src/constraints/outage_constraints.jl

```diff
@@ -95,10 +95,10 @@ end
 
 
 function add_MG_Gen_fuel_burn_constraints(m,p)
-	fuel_slope_gal_per_kwhe, fuel_intercept_gal_per_hr = generator_fuel_slope_and_intercept(
+	fuel_slope_gal_per_kwhe, fuel_intercept_gal_per_hr = fuel_slope_and_intercept(
```

Comment:
Should we do this for CHP fuel burn below as well?

Reply:
Yeah sure! Added it in two more places to eliminate dupe code